packages/net-misc/remmina/remmina.exlib

```diff
@@ -36,18 +41,19 @@ CMAKE_SRC_CONFIGURE_PARAMS=(
     -DWITH_GCRYPT=ON
     -DWITH_GETTEXT=ON
     -DWITH_LIBSECRET=ON
-    -DWITH_LIBSSH=ON
     -DWITH_TRANSLATIONS=ON
-    -DWITH_VTE=ON
-    -DWITH_ZLIB=ON
```

Comment:
why is `WITH_ZLIB` removed

Reply:
I ran cmake with the same parameters as in exlib and it warned me about unused variables: https://paste.pound-python.org/show/p2Jq2iaeXAtX5m6SpHfR/ 
zlib was cleaned from makefile somewhere around here https://github.com/FreeRDP/Remmina/pull/1263/files